### PR TITLE
Added sorting by aliases in favorites

### PR DESCRIPTION
**Pull request by stankis**

### CODE REVIEW

File: src/db/models/favorite/index.ts [hunk 1]
```diff
@@ -388,8 +389,10 @@ class Favorite extends Model {
 
         const {login} = requestedBy;
 
+        const alias = name ? name.toLowerCase() : null;
```
Comment (jhoncool) [inline src/db/models/favorite/index.ts:392]: What happened if someone send `name == ''` (empty string) to api? Is it okay that `alias` will be null and `displayAlias` will be empty string

File: src/db/scripts/update-alias-favorites.ts [hunk 2]
```diff
@@ -0,0 +1,25 @@
+/* eslint-disable camelcase */
+require('dotenv').config();
+require('../../index');
+import {db} from '../index';
+
+(async function () {
```
Comment (jhoncool) [inline src/db/scripts/update-alias-favorites.ts:6]: I think we need to write a description here about reason or use cases for that scripts.

File: src/db/models/favorite/index.ts (hunk 1 above)
Comment (stankis) [reply src/db/models/favorite/index.ts:392]: It's possible if you make api call manually (from ui it is not possible to send empty string now), anyway l agree that it should be consistent. Fixed

File: src/db/scripts/update-alias-favorites.ts (hunk 2 above)
Comment (stankis) [reply src/db/scripts/update-alias-favorites.ts:6]: I think it is a temporary script which facilitates migration, and will be deleted after release

Comment (stankis) [reply src/db/scripts/update-alias-favorites.ts:7]: Anyway, added comment with description at top

**Review by imsitnikov — DISMISSED** [review]

**Review by imsitnikov — APPROVED** [review]

**Review by imsitnikov — DISMISSED** [review]

**Review by imsitnikov — APPROVED** [review]

**Pull request merged**